minutes/meeting-20201117-5.md

```diff
@@ -0,0 +1,227 @@
+
+John_Hudson: Another idea would be a flag to have certain lookups apply to certain features after line breaking, make those options available to post-linebreak shaping. Of course existing software does it's own line breaking already, a lot of different algorithms, editors, browsers, etc. will do different line breaking even given the same text input, font, and space.
+
+nmccully: Comment about cross-script justification. There is a need for layout engines to have a user facing idea of creating a priority order for scripts in a line. Some scripts have different priorities. English=space out glyphs, scare glyphs. Perhaps when quoting Arabic words in some other script paragraph you may not want that word to stretch like it might in a native Arabic context. If there was some way to specify this it would go a long way to achieve desired results. Already running into with Korean/Japanese.
+
+John_Hudson: Question of prioritization of what happens post-linebreaking. We have the JITS table, Simon out to figure out if it was implementable, verdict was it was *almost* there but not quite. As Nat said you could specify some methods to use over others in say, Arabic. Back to prioritizing scripts over others; some of me feels this isn't quite font level information. Dependent on content of paragraph / line. Is this embed Arabic in Latin or vice versa? Already a weird gray area in various algorithms.
+
+<scribe> ACTION: Simon to introduce JSTF experiment‌ experiments.
+
+<jfkthame> knuth-plass paper: http://www.eprg.org/G53DOC/pdfs/knuth-plass-breaking.pdf
+
+reli_msft: If someone wants to do justification via the JSTF‌ table or whatere is there an approach they could follow? (apologies from scribe who can't hear well)
+
+liang: You're on for a minute.
+
+back
+
+nmccully: (justification)
+
+John_Hudson: Issue 25. Proposed axis mostly optical with one clear effect, ___‌had idea of parametric axis, not present user with 20 sliders, map parametrics to virtual axis that a user would control that mapped to multiple values in parametric axis and controlled them in unison. (scribe got lost in the forest) Part of this discussion got lost through the mess of 4 companies having a lot of impetus to get a variable system out the door, later once something shipped not so much impetus to continue innovating.
+
+liang: I'm out for a minute, sorry.
+
+spatel2: is it worth introducing a break change?
+
+liang: is the situation of multiple axes sharing the same tag (and thus allowing higher-order interpolation) providing a way forward for non-breaking changes?
+
+John_Hudson: Separate topic. We should at least document the situation in the spec and define how the behavior should be.
+
+reli_msft: avar2 would simplify the hoi situation.
+
+John_Hudson: corporate culture. I can see different companies may see breaking changes differently.
+
+Khmer encoding–shaping discussions
```

Comment:
```suggestion
### Khmer encoding–shaping discussions
```

Heading. Somehow the logging bot didn’t capture more headings. So be it then. :P